frontend/src/metabase/core/components/SelectButton/SelectButton.tsx

```diff
@@ -82,6 +82,7 @@ const SelectButton = forwardRef(function SelectButton(
         hasValue={hasValue}
         highlighted={highlighted}
         onClick={rightIcon === "close" ? handleClear : undefined}
+        style={{ flexShrink: 0 }}
```

Comment:
Fixing this icon on the fly. It was shrinking very small when the viewport got narrow